package.json

```diff
@@ -82,7 +82,7 @@
     "node-sass": "^4.7.2",
     "octicons": "^7.0.1",
     "parallel-webpack": "^2.3.0",
-    "prettier": "1.11.1",
+    "prettier": "^1.13.0",
```

Comment:
I’d recommend either using a specific version or using `~` since formatting can change in a minor release (as seen above).